packages/wp-now/README.md

```diff
@@ -4,27 +4,109 @@
 
 ## Getting Started
 
+Before getting started, make sure you have `nvm` installed. If you need to install it first,
+[follow these installation instructions](https://github.com/nvm-sh/nvm#installation).
+
 Follow these steps to build and run `wp-now` locally:
 
 ### Building
 
 To build the project, use the following commands in your terminal:
 
+1. Clone this repository locally
+2. Execute `nvm use`
+3. Execute `yarn install` followed by `yarn build`
+
+### Running
+
+1. Execute `sudo npm install -g nx@latest`. This will make `nx` command globally accessible in the project.
+2. To start the web server in the plugin or theme mode (see Modes on WP-NOW section for details), run the command below. Make sure to replace `/path/to/wordpress-plugin-or-theme` with the actual path to your plugin or theme folder.
+
 ```bash
-nvm use
-yarn install
-yarn build
+nx preview wp-now start --path=/path/to/wordpress-plugin-or-theme
 ```
 
-### Running
+### Modes on WP-NOW
+
+wp-now operates in four different modes:
+
+The mode of the `wp-now` will depend on the destination folder and whether you are working with a single plugin or a theme, `wp-content` directory or a single PHP file.
+
+-   `index`: executes a simple php file. If your destination folder does not contain a `theme`, `plugin` or `wp-content` directory, `wp-now` will run in the `index` mode serving the content of the folder without mounting a WordPress instance. For this mode, `index.php` is recommended
+-   `theme`: loads WordPress with your selected theme included
+-   `plugin`: loads WordPress with your selected plugin included
+-   `wp-content`: loads WordPress site that contains plugins and themes from the provided wp-content directory
+
+To launch the `wp-now` in the `index` mode, you can run:
+
+```bash
+nx preview wp-now start --path=/path/to/index.php-file
+```
 
-To start the web server and execute your WordPress plugin or theme, use the following command:
+To launch the `wp-now` in the `theme` or `plugin` mode, you can run:
 
 ```bash
 nx preview wp-now start --path=/path/to/wordpress-plugin-or-theme
 ```
 
-Replace `/path/to/wordpress-plugin-or-theme` with the actual path to your plugin or theme folder.
+To launch the `wp-now` in the `wp-content` mode, you can run:
+
+```bash
+nx preview wp-now start --path=/path/to/wp-content-directory
+```
+
+Make sure to replace `/path/to/wordpress-plugin-or-theme` or `/path/to/wp-content-directory` with the actual path to your plugin or theme folder or wp-content directory.
+
+### Arguments supported by wp-now start
+
+`wp-now start` currently supports the following arguments:
+
+-   `port`: the port number on which the server will listen. This is optional and if not provided, it will pick an open port number automatically;
```

Comment:
Let's list these as one would pass them, so `--port` etc. It would be good to clarify whether one needs to use `--port=8888` or `--port 8888` (unless it doesn't matter).